site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2437,6 +2439,53 @@ public function getGradedComponentsCountByGradingSections($g_id, $sections, $sec
         return $return;
     }
 
+
+    /**
+     * Returns an array of Verified components for each Section
+     *
+     * @param  array<int>  $sections
+     * @return array<int|string,int>
+     */
+    public function getVerifiedComponentsCountByGradingSections(string $g_id, array $sections, string $section_key, bool $is_team): array {
+        $unit = "users";
+        $id = "user_id";
+        if ($is_team) {
+            $unit = "gradeable_teams";
+            $id = "team_id";
+        }
+
+        $this->course_db->query(
+            "
+            SELECT 
+                " . $unit . "." . $section_key . ", COUNT(" . $unit . "." . $id . ") as verified_components_count
```

Comment:
`$section_key` is being passed into this function and directly inserted in the SQL query.  Although it looks like it can only ever be fixed strings right now, it's a security liability.  It would be better to be more defensive here and assert that `$section_key` is one of a set of expected values.